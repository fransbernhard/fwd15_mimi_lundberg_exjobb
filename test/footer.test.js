import React, { Component } from 'react';
import Footer from '../src/components/footer';
import renderer from 'react-test-renderer';

describe('components', () => {
  describe('<Footer/>', () => {
    it('renders correctly', () => {
      const tree = renderer.create(<Footer />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
