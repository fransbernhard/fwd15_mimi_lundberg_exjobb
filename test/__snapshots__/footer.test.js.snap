// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <Footer/> renders correctly 1`] = `
<footer>
  <a
    href="mailto:magdamargaretha@gmail.com"
  >
    magdamargaretha@gmail.com, 
  </a>
  <a
    href="tel:+46735139012"
  >
    +46735139012
  </a>
  <p>
    Kristianstadsgatan 7, 334 56 Malm√∂
  </p>
</footer>
`;
